docs/business-finder-email-signup.md

```diff
@@ -0,0 +1,15 @@
+# EU Exit Readiness Business Finder Email signup
+
+Currently only the [EU Exit Business Readiness Finder](https://www.gov.uk/find-eu-exit-guidance-business) is making use of these features, but other finders could use these in the future.
+
+## Subscribing to multiple facet values
+
+Users can [pick and choose which `facet_values` ('links') to subscribe to](https://github.com/alphagov/finder-frontend/pull/1191), and will be emailed when content tagged with _any of the links_ is newly tagged or updated (provided the 'send email' checkbox is checked in content tagger). For example, if a user has subscribed to updates about `Aerospace` or `EU Citizens`, and a piece of content linked only to `EU Citizens` is updated, an email will be sent.
+
+This functionality was added in [email-alert-api#867](https://github.com/alphagov/email-alert-api/pull/867) and simplified in [email-alert-api#892](https://github.com/alphagov/email-alert-api/pull/892). The functionality is described in the [email-alert-api documentation](https://github.com/alphagov/email-alert-api/blob/master/doc/matching-content-to-subscriber-lists.md).
```

Comment:
I don't think this section is needed. I think just a link saying see https://github.com/alphagov/email-alert-api/blob/master/doc/matching-content-to-subscriber-lists.md for more details is sufficient.